keps/sig-scheduling/1819-scheduler-extender/kep.yaml

```diff
@@ -0,0 +1,27 @@
+title: Scheduler Extender
+kep-number: 1819
+authors:
+  - "@ravigadde"
```

Comment:
the original author is @ravigadde, see the first [proposal and implementation PR](https://github.com/kubernetes/kubernetes/pull/13580)

Reply:
we should add a note that this KEP was done in retrospect and it was based on the original extender proposal.

Reply:
add the note here or "Summary" section in README.md?

Reply:
perhaps a note at the end of the summary second.

Reply:
done